docker/build/go.mod.template

```diff
@@ -0,0 +1,8 @@
+module processor_native_src
+
+go 1.12
+
+require (
+    github.com/orbs-network/orbs-contract-sdk v1.1.0
```

Comment:
if possible add comments indicating what each version means and how this file is used